apps/charterafrica/src/components/EcosystemList/useEcosystemList.js

```diff
@@ -5,9 +5,13 @@ import fetchJson from "@/charterafrica/utils/fetchJson";
 const fetcher = (url) => fetchJson.get(url);
 
 function useEcosystemList(params) {
-  const { collection, ...rest } = params;
-  const searchParams = new URLSearchParams(rest).toString();
-  const key = `/api/v1/resources/collections/${collection}${
+  const param = new URLSearchParams();
+  Object.keys(params).forEach((key) => {
+    if (params[key] && params[key] !== "undefined")
+      param.append(key, params[key]);
+  });
+  const searchParams = param.toString();
+  const key = `/api/v1/resources/collections${
```

Comment:
1. See how `useDocuments`, `useDatasets`, `useArticles`, etc. have been written.
2. Isn't `useEntities` a better name or is this something completely different?